README.md

```diff
@@ -1,26 +1,46 @@
-# Package Managers Special Interest Group
+![Package Managers Special Interest Group](https://github.com/ipfs/package-managers/blob/master/packagemanagers-sig-header.png "Package Managers Special Interest Group")
 
 [![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](https://protocol.ai/)
 [![](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/)
 [![](https://img.shields.io/badge/freenode-%23ipfs--package--managers-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs-package-managers)
 
+## Join our weekly sync (all welcome!)
+Tuesdays at 3:30pm UTC / 4:30pm BST / 7:30am PST
+
+📞 Zoom: https://protocol.zoom.us/j/5011724290
+
+## Why are package managers important to the future of IPFS?
+
+- IPFS is *very close* to becoming a great tool to solve real problems for package manager users, package publishers, and package manager maintainers! Plus, closing those gaps will make IPFS better for everyone while giving the IPFS community a yardstick for how to measure our success in the future.
+- Package manager maintainers and consumers are an important demographic to IPFS' future: developers who may not be IPFS pros yet, but who can get excited and engaged about IPFS in its current state, enabling everyone to help IPFS improve both on existing fronts and in problem areas we haven't yet envisioned
+- Introducing IPFS tooling and support can create direct value and impact in the overall package manager ecostystem by saving developers time and empowering resilient development experiences for developers using IPFS or IPFS-powered tools and registries.
+
+## Goals for IPFS and package managers
+1. Get the package manager maintainer community excited about the benefits of using IPFS in their ecosystems
+2. Demonstrate to package consumers that IPFS is _holistically better_ than current centralized package managers (this may mean parity on some use cases, but significantly better on others)
```

Comment:
```suggestion
2. Demonstrate to package consumers that IPFS is _holistically better_ than current centralized package managers (this may mean parity on some use cases, but significantly better on others)
3. Increase awareness and engagement with IPFS (either to build new tools, or contribute directly) among package manager users / package publishers
```

I think another goal is looping in the community of package users / developers

Reply:
Good call, thank you!